Dockerfile

```diff
@@ -0,0 +1,60 @@
+#########
+FROM node:18-alpine as frontend-builder
+
+# Set the working directory to /app inside the container
+WORKDIR /app
+
+# Copy source files
+COPY packages/frontend .
+
+# Install dependencies (npm ci makes sure the exact versions in the lockfile gets installed)
```

Comment:
`npm ci`?

Reply:
This comment was taken from another dockerfile template, the vivo frontend uses `yarn`, not `npm`.